lib/libzpool/util.c

```diff
@@ -174,8 +174,9 @@ set_global_var_parse_kv(const char *arg, char **k_out, u_longlong_t *v_out)
 		goto err_free;
 	}
 
-	*k_out = k;
+	*k_out = strdup(k);
 	*v_out = val;
+	free(d);
 	return (0);
 
 err_free:
```

Comment:
Right after this line is a free(k) that should be free(d). This is a note to myself to revise this in the morning